onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3408,10 +3413,19 @@ def aten_negative(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_new_empty(self: TensorType, size: INT64) -> TensorType:
+@torch_op("aten::new_empty")
+def aten_new_empty(self, size: INT64, dtype: int = -1) -> TensorType:
+    """new_empty.
+
+    Note: dtype is an onnx enum. Users should convert torch dtype to onnx dtype
+    before calling this function.
+    """
     # new_empty(Tensor self, SymInt[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
 
-    raise NotImplementedError()
+    vals = np.empty(shape=size).flatten()
```

Comment:
All of operations within this function will be transferred to ONNX finally. I think it could not handle numpy functions. 

How about op.Flatten()?